src/mugs.js

```diff
@@ -468,6 +468,20 @@ define([
         }
     });
 
+    Object.defineProperty(Mug.prototype, "hashtagPath", {
```

Comment:
I think it makes sense to document that list in a comment here and try to keep it updated if/as things change.